ecf/scripts/workflow_manager/cycled/next_run.ecf

```diff
@@ -0,0 +1,58 @@
+#PBS -S /bin/bash
+#PBS -N %RUN%_next_run_%CYC%
+#PBS -j oe
+#PBS -q %QUEUE%
+#PBS -A %PROJ%-%PROJENVIR%
+#PBS -l walltime=00:30:00
+#PBS -l place=vscatter,select=1:ncpus=1:mem=1GB
```

Comment:
Please put `place` and `select` on separate lines as is the case in all the `ecf` job cards.

Reply:
Ok